code/datums/gamemodes/gm_parent.dm

```diff
@@ -262,6 +251,17 @@
 						stuff_to_output += "<B>[traitor_name] was a [string]!</B>"
 		catch(var/exception/e)
 			logTheThing("debug", null, null, "kyle|former-antag-runtime: [e.file]:[e.line] - [e.name] - [e.desc]")
+	
+	// Display all antagonist datums. We arrange them like this so that each antagonist is bundled together by type
+	for (var/V in concrete_typesof(/datum/antagonist))
+		var/datum/antagonist/dummy = V
+		for (var/datum/antagonist/A as anything in get_all_antagonists(initial(dummy.id)))
```

Comment:
why initial

Reply:
I seem to recall that `initial()` can fetch the default value of provided typepath even without an instance, which I'm doing here since I don't believe that `concrete_typesof` returns instances, and as such, the values can't have variables directly referenced. is this true?

Reply:
yeah idk why i commented that i realized it while reviewing lol